techniques/general/G99.html

```diff
@@ -2,98 +2,80 @@
-            they relate to the normative WCAG 2.1 success criteria. The Applicability section
-            explains the scope of the technique, and the presence of techniques for a specific
-            technology does not imply that the technology can be used in all situations to create
-            content that meets WCAG 2.1.
+         <h2>達成方法に関する重要な情報</h2>
+         <p>これらの達成方法 (参考) の使用法及び、それらが WCAG 2.1 達成基準 (規定) とどのように関係するかに関する重要な情報については、<a href="https://waic.jp/docs/WCAG21/Understanding/understanding-techniques.html">WCAG 達成基準の達成方法を理解する</a>を参照のこと。適用 (対象) セクションは、その達成方法の範囲について説明しており、特定の技術に関する達成方法があるからといって、WCAG 2.1 を満たすコンテンツを作成する際に、常にその技術が使用可能であるわけではない。
          </p>
       </section>
       <main>
          <section id="applicability">
-            <h2>Applicability</h2>
-            <p>Content where user actions cause content to be deleted.</p>
-            <p>This technique relates to <span><a href="https://w3c.github.io/wcag/understanding/error-prevention-legal-financial-data">Success Criterion 3.3.4: Error Prevention (Legal, Financial, Data)</a> (Sufficient)</span>.
+            <h2>適用 (対象)</h2>
+            <p>利用者の操作によって削除されるコンテンツ</p>
+            <p>これは<span><a href="https://w3c.github.io/wcag/understanding/error-prevention-legal-financial-data">達成基準 3.3.4: エラー回避 (法的、金融、データ)</a> (十分な達成方法)</span> に関する達成方法である。
             </p>
          </section>
          <section id="description">
-            <h2>Description</h2>
-            <p>When a Web application provides the capability of deleting information, the server
-               can provide a means to recover information that was deleted in error by a user. One
-               approach is to delay deleting the data by merely marking it for deletion or moving
-               it to a holding area (such as a trash can) and waiting some period of time before
-               actually deleting it. During this time period, the user can request that the data
-               be restored or can retrieve it from the holding area. Another approach is to record
-               all delete transactions in such a way that data can be restored if requested by the
-               user, such as in the edit history stored by wikis and source control applications.The
-               retrievable information that is stored should be that which would be needed to correct
-               the transaction.
+            <h2>解説</h2>
+            <p>この達成方法の目的は、ウェブアプリケーションで情報を削除できる場合、利用者によって誤って削除された情報を復元する手段をサーバーが提供することである。一つの方法として、削除をするものにマークを付けるだけにする、又は (ゴミ箱のように) 情報を保持する領域に移動させるなど、データの削除に時間差を設けて、実際に削除するまでにしばらくの間待機することが考えられる。この間、利用者はデータを復元したり、一時的な保持領域から取り出したりすることができる。他には、wiki 及びソースコントロールアプリケーションで編集履歴が保持されているように、利用者が必要なときにデータを復元できるように全ての削除操作を記録しておく方法がある。トランザクションの修正が必要となる情報は保存し、利用者が取り出し可能なようにしておくべきである。
```

Comment:
```suggestion
            <p>ウェブアプリケーションが情報を削除する機能を提供する場合、サーバーは利用者によって誤って削除した情報を復元する手段を提供できる。一つの方法は、削除のために単にマークを付ける、又は (ゴミ箱のように) データの保持領域に移動させ、実際に削除するまでにしばらくの間待機することによって、データの削除を遅らせることである。この間、利用者はデータの復元を要求したり、保持領域から取り出したりすることができる。他には、wiki 及びソースコントロールアプリケーションで保存される編集履歴のように、利用者が要求した場合にデータを復元できるような方法で、全ての削除トランザクションを記録しておく方法がある。保存される検索可能な情報は、トランザクションの修正に必要なものであるべきである。
```

Reply:
```suggestion
            <p>ウェブアプリケーションが情報を削除する機能を提供する場合、サーバーは利用者によって誤って削除した情報を復元する手段を提供できる。一つの方法は、削除のために単にマークを付ける、又は (ゴミ箱のように) データの保持領域に移動させ、実際に削除するまでにしばらくの間待機することによって、データの削除を遅らせることである。この間、利用者はデータの復元を要求したり、保持領域から取り出したりすることができる。他には、wiki 及びソースコントロールアプリケーションで保存される編集履歴のように、利用者が要求した場合にデータを復元できるような方法で、全ての削除トランザクションを記録しておく方法がある。保存される取得可能な情報は、トランザクションの修正に必要なものであるべきである。
```

Reply:
最後のセンテンス、「保存される**検索**可能な情報は、...」は、「The **retrievable** information that is stored should be ...」なので、「保存される**取得**可能な情報は、...」としてみました。

Reply:
@caztcha コメントありがとうございます。

[retrieval](https://ejje.weblio.jp/content/retrieval) を考えると、「回復」とあるので、文脈として**回復可能**が適当かなあと。

```suggestion
            <p>ウェブアプリケーションが情報を削除する機能を提供する場合、サーバーは利用者によって誤って削除した情報を復元する手段を提供できる。一つの方法は、削除のために単にマークを付ける、又は (ゴミ箱のように) データの保持領域に移動させ、実際に削除するまでにしばらくの間待機することによって、データの削除を遅らせることである。この間、利用者はデータの復元を要求したり、保持領域から取り出したりすることができる。他には、wiki 及びソースコントロールアプリケーションで保存される編集履歴のように、利用者が要求した場合にデータを復元できるような方法で、全ての削除トランザクションを記録しておく方法がある。保存される回復可能な情報は、トランザクションの修正に必要なものであるべきである。
```

Reply:
なるほど、「回復可能」ですね。異存ありませんので、コミットさせていただきます。